src/events/sql.js

```diff
@@ -37,7 +37,9 @@ module.exports.wrapSqlQuery = function wrapSqlQuery(queryString, params, callbac
     try {
         let sqlObj = {};
         try {
-            sqlObj = sqlParser.parse(queryString);
+            // Sanitizing query.
+            const queryStringSan = queryString.split('`').join('').replace(';', '');
```

Comment:
Why not just `queryString.replace('`', '')`?
And why do you want to remove ';'? It used to concatenate several queries

Reply:
replace in js just replaces the first instance :(